addons/website/static/src/xml/website.cookies_bar.xml

```diff
@@ -7,25 +7,37 @@
     </t>
     <t t-name="website.cookies_bar.text_primary">
         <p class="o_cookies_bar_text_primary">
-            We use cookies to provide you a better user experience.
+            Allow the use of cookies from this website on this browser?
         </p>
     </t>
     <t t-name="website.cookies_bar.text_secondary">
         <p class="o_cookies_bar_text_secondary">
-            We use them to store info about your habits on our website. It will helps us to provide you the very best experience and customize what you see. <br/>
-            By clicking on this banner, you give us permission to collect data.
+            We use cookies to provide improved experience on this website. You can learn more about our cookies and how we use them in our <a href="/cookie-policy" class="o_cookies_bar_text_policy">Cookie Policy</a>.
         </p>
     </t>
+    <t t-name="website.cookies_bar.text_button_all">
+        <a href="#" id="cookies-consent-all" role="button"
+           class="js_close_popup o_cookies_bar_accept_all o_cookies_bar_text_button btn btn-outline-primary rounded-circle mb-1 px-2 py-1">Allow all cookies</a>
+    </t>
+    <t t-name="website.cookies_bar.text_button_essential">
+        <a href="#" id="cookies-consent-essential" role="button"
+           class="js_close_popup o_cookies_bar_accept_essential o_cookies_bar_text_button_essential btn btn-outline-primary rounded-circle mt-1 mb-2 px-2 py-1">Only allow essential cookies</a>
+    </t>
     <t t-name="website.cookies_bar.discrete">
+        <!-- When updating this template, do not forget to also update the
+             backend `website.cookies_bar` template -->
         <section class="o_colored_level o_cc o_cc1">
             <div class="container">
-                <div class="row">
-                    <div class="col-lg-8 pt16">
-                        <p>We use cookies to provide you a better user experience.</p>
+                <div class="row pt16 pb16">
```

Comment:
The padding on row can't be edited through the editor, that's why it was on the col, could you please fix that in another pr, so it doesn't block this one?

Reply:
@rdeodoo Moved padding back to column in https://github.com/odoo/odoo/pull/101859.